lib/commands/network.js

```diff
@@ -48,18 +44,7 @@ export async function setWifiState (on, isEmulator = false) {
 export async function setDataState (on, isEmulator = false) {
   if (isEmulator) {
     // The svc command does not require to be root since API 26
-    await this.adb.shell(['svc', 'data', on ? 'enable' : 'disable'], {
-      privileged: await this.adb.getApiLevel() < 26,
-    });
-    return;
-  }
-
-  if (await this.adb.getApiLevel() < 30) {
-    await this.checkBroadcast([
-      '-a', DATA_CONNECTION_SETTING_ACTION,
-      '-n', DATA_CONNECTION_SETTING_RECEIVER,
-      '--es', 'setstatus', on ? 'enable' : 'disable'
-    ], `${on ? 'enable' : 'disable'} data`);
+    await this.adb.shell(['svc', 'data', on ? 'enable' : 'disable']);
     return;
```

Comment:
return is redundant

Reply:
https://github.com/appium/io.appium.settings/pull/243/commits/731541c62c8aaf255806c9185017035dd4c246ed